tests/constants/urls.js

```diff
@@ -8,6 +8,7 @@ export const APPS = {
   TASKS: 'https://my.realdevsquad.com/tasks',
   IDENTITY: 'https://my.realdevsquad.com/identity',
   MY_STATUS: 'https://my.realdevsquad.com',
+  API_BACKEND: 'https://staging-api.realdevsquad.com',
```

Comment:
Why this present here? Where is it being used?
And what and where all APP is being used?

Reply:
It is being used to test if auth url is generated properly in login-test.js